code/extensions/MultiFormObjectDecorator.php

```diff
@@ -14,43 +14,45 @@
  *
  * @package multiform
  */
-class MultiFormObjectDecorator extends DataExtension {
+class MultiFormObjectDecorator extends DataExtension
+{
+    private static $db = array(
+        'MultiFormIsTemporary' => 'Boolean',
+    );
 
-	private static $db = array(
-		'MultiFormIsTemporary' => 'Boolean',
-	);
+    private static $has_one = array(
+        'MultiFormSession' => 'MultiFormSession',
+    );
 
-	private static $has_one = array(
-		'MultiFormSession' => 'MultiFormSession',
-	);
+    public function augmentSQL(SQLSelect $query)
```

Comment:
this is a formatting PR not one for fixes :P that'll need a different PR to sort out